index.html

```diff
@@ -959,11 +971,9 @@ <h2 id="arrays">Array Functions</h2>
         If duplicate keys exist, the last value wins.
       </p>
       <pre>
-_.object(['moe', 'larry', 'curly'], [30, 40, 50]);
-=&gt; {moe: 30, larry: 40, curly: 50}
+_.unzip(["moe", 30, true], ["larry", 40, false], ["curly", 50, false]);
```

Comment:
Why is the unzip example below the `_.object` documentation? And why did you get rid of the _.object example?